spec/feature/cards_managment.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+feature "Editing cards" do
+  scenario "Try to edit a card" do
+    create(:card)
+    visit root_path
+
+    click_link "Карточки"
+    click_link "Редактировать", match: :first
+
+    fill_in "card_original_text", with: "Лошадь"
+    fill_in "card_translated_text", with: "Horse"
+    click_button("Изменить")
+    expect(page).to have_content  "Лошадь"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.